cura/CuraActions.py

```diff
@@ -181,5 +185,50 @@ def setBuildPlateForSelection(self, build_plate_nr: int) -> None:
 
         Selection.clear()
 
+    @pyqtSlot()
+    def cut(self) -> None:
+        self.copy()
+        self.deleteSelection()
+
+    @pyqtSlot()
+    def copy(self) -> None:
+        mesh_writer = cura.CuraApplication.CuraApplication.getInstance().getMeshFileHandler().getWriter("3MFWriter")
+
+        # Get the selected nodes
+        selected_objects = Selection.getAllSelectedObjects()
+        # Serialize the nodes to a string
+        scene_string = mesh_writer.sceneNodesToString(selected_objects)
+        # Put the string on the clipboard
+        QApplication.clipboard().setText(scene_string)
+
+    @pyqtSlot()
+    def paste(self) -> None:
+        application = cura.CuraApplication.CuraApplication.getInstance()
+
+        # Parse the scene from the clipboard
+        scene_string = QApplication.clipboard().text()
+
+        mesh_reader = application.getMeshFileHandler().getReaderForFile(".3mf")
```

Comment:
Derp. I didn't notice this the previous time. But you can actually disable 3mf reader

If the user did this, there won't be mesh_reader returned here.

Reply:
You could make the 3mf reader a required plugin, or you could disable copy/paste if the plugin is not available.

Reply:
https://github.com/Ultimaker/Cura/pull/16383/commits/c393d915d7e85e7437846475d416ead9c3455c0d